contrib/slime-repl.el

```diff
@@ -1017,15 +1034,21 @@ If EXCLUDE-STRING is specified then it's excluded from the search."
   "Cycle backwards through input history.
 If the `last-command' was a history navigation command use the
 same search pattern for this command.
-Otherwise use the current input as search pattern."
+Otherwise use the current input as search pattern.
+
+With a prefix-arg, do replacement from the mark."
   (interactive)
-  (slime-repl-history-replace 'backward (slime-repl-history-pattern t)))
+  (let ((slime-repl-history-use-mark current-prefix-arg))
```

Comment:
Since `slime-repl-history-use-mark` is unconditionally bound here, setting it to `t` has no effect, right?

Reply:
Right 🤦‍♂️ thanks for catching that.

Are you happy with the behaviour outside of that fault?

Reply:
Yes, WFM.